lib/commons/dom/is-hidden-with-css.js

```diff
@@ -10,8 +11,10 @@ import { getNodeFromTree } from '../../core/utils';
  * @param {Boolean} descendentVisibilityValue (Optional) immediate descendant visibility value used for recursive computation
  * @return {Boolean} the element's hidden status
  */
-function isHiddenWithCSS(el, descendentVisibilityValue) {
-  const vNode = getNodeFromTree(el);
+function isHiddenWithCSS(node, descendentVisibilityValue) {
```

Comment:
This change isn't necessary once the isVisible refactor is in (the function name changed and it already handles virtual nodes). Do we still want to make this change if `isHiddenWithCSS` is being deprecated and not used anymore in our code?

Reply:
I didn't want to create a dependency on your refactor. I don't think doing both is a problem.